src/Invio.Immutable/SinglePropertyHandler.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Reflection;
+
+namespace Invio.Immutable {
+    /// <summary>
+    ///   Performs limited precision comparisons an hash code generation for properties of type
+    ///   <see cref="Single" />.
+    /// </summary>
+    public sealed class SinglePropertyHandler : PropertyHandlerBase<Single> {
+        /// <summary>
+        ///   The level of precision to use when comparing floating point values.
+        /// </summary>
+        public Int32 Precision { get; }
+
+        /// <summary>
+        ///   Indicates whether precision should be interpreted as the number of signficant figures,
+        ///   or as the number of signficants digits to the right of the decimal place.
+        /// </summary>
+        public Boolean SignificantFigures { get; set; }
```

Comment:
I get why this would be mutable on the Attribute, but I don't see why this should be able to be mutated on the handler. Was this a simple oversight, or was there a specific reason for making this mutable?

Reply:
yup, oversight.